packages/site/src/components/DocumentOutline.tsx

```diff
@@ -60,11 +60,14 @@ const Headings = ({ headings, activeId }: Props) => (
           href={`#${heading.id}`}
           onClick={(e) => {
             e.preventDefault();
-            const el = document.querySelector(`#${heading.id}`);
+            const el = document.querySelector(`#${heading.id}`) as HTMLElement | undefined;
             if (!el) return;
 
             el.scrollIntoView({ behavior: 'smooth' });
             history.replaceState(undefined, '', `#${heading.id}`);
+            // Changes keyboard tab-index location
+            if (el.tabIndex === -1) el.tabIndex = -1;
```

Comment:
@rowanc1 did you want to change the tab index?

Reply:
This actually does, -1 is the `get` default, but it isn't actually set. When you set it to -1 that actually changes the behaviour in the browser!

Reply:
Ah, I see!